packages/destination-actions/src/destinations/fullstory/__tests__/fullstory.test.ts

```diff
@@ -43,6 +121,7 @@ describe('FullStory', () => {
       })
 
       expect(response.status).toBe(200)
+      // TODO(nate): Consider using .toEqual for full deep equality verification
```

Comment:
I'm planning to open a fast follow PR that cleans up artifacts like this from earlier iterations. Omitted here to keep PR surface smaller and more focused on the new `trackEvent` action.